components/partials/header.js

```diff
@@ -26,10 +26,10 @@ export default function Header() {
   return (
     <>
       <header
-        className={`sticky w-full top-0 z-10 bg-white ${
+        className={`w-full top-0 z-10 bg-white ${
           showHeader
-            ? `visible ${lastScrollPos > 0 ? "drop-shadow-lg" : ""}`
-            : "invisible"
+            ? `sticky ${lastScrollPos > 0 ? "drop-shadow-lg" : ""}`
+            : "relative"
         }`}
```

Comment:
i have change this becoz i was see error on header and i was get solution this error ,so i have slove the error.